tests/cli/workflow/taxprofiler/test_cli_taxprofiler_compound_commands.py

```diff
@@ -3,7 +3,7 @@
 from _pytest.logging import LogCaptureFixture
 from click.testing import CliRunner
 
-from cg.cli.workflow.taxprofiler.base import start, start_available, taxprofiler
+from cg.cli.workflow.taxprofiler.base import start, start_available, taxprofiler, store
```

Comment:
store and CgError are not used

We have some pre-commit hooks to check for these kind of things FYI: https://atlas.scilifelab.se/infrastructure/tech_stack/editors/pycharm/#install-pre-commit

Reply:
Thanks, I was not aware of this one :)